app/utils/whitelabel.ts

```diff
@@ -32,8 +32,8 @@ interface WhiteLabelSite {
 
 // Include a domain in config.js
 function determineWhiteLabelSite(): WhiteLabelSiteKey {
-  if (window.location.host === config.SFFAMILIES_DOMAIN) return 'SFFamilies';
-  if (window.location.host === config.MOHCD_DOMAIN) return 'SFServiceGuide';
+  if (window.location.host.split('.')[0] === String(config.SFFAMILIES_DOMAIN) || window.location.host.split('.')[0] === `${String(config.SFFAMILIES_DOMAIN)}-staging`) return 'SFFamilies';
+  if (window.location.host.split('.')[0] === String(config.MOHCD_DOMAIN) || window.location.host.split('.')[0] === `${String(config.MOHCD_DOMAIN)}-staging`) return 'SFServiceGuide';
```

Comment:
This makes the line very long and hence more difficult to read on GitHub or even in a text editor. Could you factor the check into a helper function?

Reply:
Yea it's not pretty, I made it a little shorter for now.